CHANGES.md

```diff
@@ -1,6 +1,7 @@
 pytest-watch Changelog
 ======================
 
+- Enhancement: Use the same Python interpreter for `py.text` when `ptw` is run outside of an activated virtual environment.
```

Comment:
Good catch, @blueyed! I completely missed that before.

Patched this up a0fff61168e0555e66d8aad4eee8f61c385e6b80.